components/ai_chat/resources/page/state/active_chat_context.tsx

```diff
@@ -103,17 +103,17 @@ function ActiveChatProvider({ children, selectedConversationId, updateSelectedCo
 
   // Handle the case where a non-existent chat has been selected:
   React.useEffect(() => {
-    // We can't tell if an id is valid until we've loaded the list of visible
+    // We can't tell if an id is valid until we've loaded the list of non-empty
     // conversations.
     if (!initialized) return
 
     // Special case the default conversation - it gets treated specially as
     // the chat is rebound as the tab navigates.
     if (selectedConversationId === tabAssociatedChatId) return
     if (!selectedConversationId) return
-    if (visibleConversations.find(c => c.uuid === selectedConversationId)) return
+    if (conversationMetas.find(c => c.uuid === selectedConversationId)) return
```

Comment:
@petemill I'm not sure about this case, should this be limited to conversations with has_content = true?

Reply:
It is fine as you have it here - it's valid in terms of this module as a conversation will be found and rendered successfully.